WeakAurasTemplates/TriggerTemplates.lua

```diff
@@ -133,44 +252,72 @@ local function buildCondition(trigger, check, properties)
   return result;
 end
 
-local function buffGreyed(conditions, trigger)
-  tinsert(conditions, buildCondition(trigger, checks.buffedFalse, {changes.grey}));
+local function missingBuffGreyed(conditions, trigger, regionType)
+  if regionType ~= "model" then
```

Comment:
There is the possibility of adding region types via plugins, and in fact there's a plugin: StopMotion

So any check for regionTypes needs to test the supported ones. Testing for not model does not work.

See the later comment on where to move the check though.

Reply:
done in #664 

To check if the property exists for the regionType i did this:
WeakAuras.regionTypes[regionType].default[property] 

I'm not 100% sure this is the right way to test it.